SimMuon/Configuration/python/customiseForMuonAging.py

```diff
@@ -0,0 +1,24 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
Hi @battibass - so its agreed to move forward given the production plans agreed already. 

this needs to small updates 
1) isChosen is deprecated, so needs to be removed.I would suggest just removing the if and supporting only the upgrade here.
2) please add some documentation to this file explaining that its use removes reproducibility from the reco sequence.